store/src/java/com/zimbra/cs/mailbox/Mailbox.java

```diff
@@ -924,7 +910,7 @@ public String getAccountId() {
     }
 
     public MailboxData getData() {
-        return mData;
+        return mData; //TODO: should this return the latest state synced with Redis?
```

Comment:
@ilyusha Looks like the only code that I saw that is using this is MailItem and the only things it is looking up from that is the mailbox id and account id.  I think is OK as is.

Reply:
thanks for looking into that!